cita-network/src/mq_agent.rs

```diff
@@ -0,0 +1,213 @@
+// CITA
+// Copyright 2016-2019 Cryptape Technologies LLC.
+
+// This program is free software: you can redistribute it
+// and/or modify it under the terms of the GNU General Public
+// License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any
+// later version.
+
+// This program is distributed in the hope that it will be
+// useful, but WITHOUT ANY WARRANTY; without even the implied
+// warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE. See the GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::network::{LocalMessage, NetworkClient};
+use crate::node_manager::{BroadcastReq, NodesManagerClient};
+use crossbeam_channel::{unbounded, Receiver, Sender};
```

Comment:
This will be a problem in future.

The channel send to [`fn start_pubsub()`] as input, should be same as the [definitions].
So, the changes in upstream (crate _`pubsub`_) will not effect downstream (code, here).

[`fn start_pubsub()`]: https://github.com/cryptape/cita-common/blob/8c25204/pubsub/src/lib.rs#L39-L78
[definitions]: https://github.com/cryptape/cita-common/blob/8c25204/pubsub/src/lib.rs#L26-L36

Reply:
Yes, This should be the same define as `fn start_pubsub()`, I will use `pubsub::channel` instead.